src/components/alert/scripts/build-dev.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ -z "$VARIANT" ]; then VARIANT=''; fi
+if [ -z "$TEMPLATE_LANG" ]; then TEMPLATE_LANG='en'; fi
+
+echo "Building package for demo. Variant: '$VARIANT', language: '$TEMPLATE_LANG'."
+
+browserify -t [ redirectify --dir '$VARIANT' ] -t reactify dev/example.jsx -o dev/app.js
```

Comment:
how is this script used? I want to run it to check that '$VARIANT' is doing what we expect... if you call
`VARIANT=foo build-dev.sh`
it looks like it should pass the value `foo` to redirectify but I think it might actually send `$VARIANT`... let me know how this script is used and I will check. Looks like this little script doesn't do enough to justify its existence. It doesn't seem to do anything with the value of TEMPLATE_LANG either.